.github/workflows/release-sandbox.yml

```diff
@@ -30,7 +30,7 @@ env:
 
 jobs:
   release:
-    runs-on: [self-hosted, epinio]
+    runs-on: ubuntu-latest
```

Comment:
https://github.com/epinio/epinio/actions/runs/4957350161/jobs/8868917727?pr=2297#step:10:14

Indicates a change in the underlying shell, one which does not understand the `-o pipefail`.
Suspect that the release-publish flow may run into the same.
Might be fixable to changing the script to work without `-o pipefail`.

Reply:
yes realized the same - was reading about problems related to /bin/sh vs. bash in ubuntu - suggesting to run the script (executable) with "sudo" ...

Reply:
yep, changing "#!/bin/sh" to "#!/bin/bash" did it.